requirements/specifications/intents/user-interest.md

```diff
@@ -0,0 +1,299 @@
+| ------ | ----------- |
+| `playlist` | Interested in adding to a list |
+| `reaction` | Interested in submitting a reaction, e.g. like or dislike |
+| `recording` | Interest in scheduling a recording |
+| `share` | Interest in sharing the content on social media |
+
+**NOTE**: We can remove some of these (not `playlist`) these are here for now to illustrate the purpose for the reason paramater.
+
+An app **MUST** `provide` the `xrn:firebolt:capability:discovery:interest`
+capability in order to call `Discovery.userInterest`.
+
+When this method is called with a valid `EntityDetails`, the platform
+**MUST** dispatch a `Content.onUserInterest` notification to all Apps
+that have registered for it (typically Aggregated Experience Apps) with
+information about the app, interest type, and the entity.
+
+The `Content.onUserInterest` event has three context parameters:
+
+| context | type | description |
+|---------|------|-------------|
+| appid   | string | The id of the app that pushed the user interest. |
+| type    | `InterestType` | the type of interest. |
+| reason  | `InterestReason` | the reason for the interest |
+
+The `Content.onUserInterest` event has a result type of `EntityDetails`.
+
+An Aggregated Experience can register for the `Content.onUserInterest`
+notification, and it will receive notifications when an `EntityDetails` is
+returned from the active App after a `Discovery.userInterest` call is
+fulfilled.
+
+An app **MUST** have permissions to `use` the
+`xrn:firebolt:capability:discovery:interest` capability in order to 
+listen to the `Content.onUserInterest` notification.
+
+If the result is `null` or is not a valid entity, i.e. does not match
+the [EntityDetails](../entities/) schema, then no `Content.onUserInterestedIn`
+notification will be dispatched.
+
+The `Discovery.userInterest` method **SHOULD NOT** be used in place of more
+specific Discovery methods, e.g. `Discovery.watchNext` or
+`Discovery.watched`. These methods facilitate specific UX flows that may
+have separate legal opt-outs for each user.
+
+The `Discovery.userInterest` method **SHOULD NOT** be called unless the user
+is activating a UI element in the app, or in a second screen experience
+that is communicating with the app, that implies interest of some kind.
+
+### 3.2. User Interest from a platform UX
+
+Firebolt platforms may provide a platform UX, e.g. voice or and RCU, to
+express user interest in content from an active App. To facilitate this
+Apps will need to be told about the user's expressed interest in their
+content.
+
+First, the Aggregated Experience (or some app with this
+capability) detects that the user is interested in something. In this
+picture the interest is triggered by an RCU button, but how this occurs
+is outside the scope of this document. When this happens, the Aggregated
+Experience app calls `Content.requestUserInterest()`, which will trigger the
+foreground app's UserInterest provider and call it's `userInterest` method
```

Comment:
This is configurable per distributor, right? Not a requirement for Firebolt to only invoke the foreground provider?

Reply:
yeah, and it's based on focus now, anyway.

Reply:
oops, got lazy. it was resolved in my mind :)

fixed.